fh-messaging/lib/services/mbaas_metrics.js

```diff
@@ -79,6 +79,7 @@ module.exports = function(mongodb) {
   }
 
   function getSingleMbaasMetricsCollectionName(metricType) {
+    console.log('@@@getting mbaas metric', "mbaas_" + metricType);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)